### PR TITLE
Add about application modal with username and active app

### PR DESCRIPTION
**Pull request by karelhala**

### CODE REVIEW

**Comment by karelhala** [comment]
@iphands I've added basic info like username and which app is active. I am plannig to add frontend components version, chrome version, browser info and OS info. What else do you think might be good to add here?

**Comment by ryelo** [comment]
@karelhala following up on the conversation earlier, let's move the footer information into the about modal. Also, what are your thoughts about moving the account_number from the user dropdown and moving it into the modal?

**Comment by karelhala** [comment]
>  moving the account_number from the user dropdown and moving it into the modal

I'd keep it in user dropdown, but let's add it to about modal as well

File: src/js/App/Header/InsightsAbout.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            if (app.subItems) {
+                const subApp = app.subItems.find(subItem => subItem.id === activeApp);
+                app = subApp && subApp.reload ? subApp : app;
+            }
+        }
+        return (
+            <AboutModal
+                isOpen={isModalOpen}
+                onClose={onClose}
+                productName="Redhat Insights"
```
Comment (ryelo) [inline src/js/App/Header/InsightsAbout.js]: Red Hat - two words

**Comment by iphands** [comment]
@karelhala https://github.com/RedHatInsights/insights-chrome/tree/features/enable-app-about
I linted, fixed "Redhat" and started refactoring a little. I would like to merge this and quickly add the  app.info.json stuff.

Will work on that today barring some other :fire:

**Comment by ryelo** [comment]
Should we close this PR and open it against the `enable-app-about` branch on this repo?

**Comment by karelhala** [comment]
Yeah, this can be closed and @iphands can follow in `enable-app-about` I thought I'd be much faster with this. But don't have much of a time to continue with it.

**Pull request closed without merging**